src/scenes/gameplay-scene.ts

```diff
@@ -484,7 +485,7 @@ export class GameplayScene {
       this.stoneHandler.getCorrectTargetStone() == this.tempWordforWordPuzzle &&
       isCorrect
     ) {
-      this.handleCorrectStoneDrop(feedBackIndex);
+      this.handleCorrectStoneDrop(feedBackIndex); //handle what text
```

Comment:
@bernhardccodev can you add proper comment here on this method or you can remove the existing comment as it is not that clear.